subsys/bluetooth/host/Kconfig

```diff
@@ -298,6 +298,16 @@ config BT_SMP_SC_ONLY
 	  Security Mode 1 Level 4 stands for authenticated LE Secure Connections
 	  pairing with encryption. Enabling this option disables legacy pairing.
 
+config BT_SMP_UNAUTH_REQUIRE_UNPAIR
+	bool "Disallow unauthenticated pairing for paired device"
```

Comment:
Would it make sense to enable this by default? Generally "secure by default" is a good policy.

Reply:
Yep, I agree with that one too.

Reply:
I agree too, the only thing I'm wondering if I should have the option at all.
I guess in case someone is relying on this behavior and find it acceptable.

Reply:
We could consider inverting the option in that case, i.e. `=n` means "secure". Perhaps that would give a more understandable name?

Reply:
Good point. I'll switch from `NO` to `ALLOW` then.